README.md

```diff
@@ -93,6 +99,14 @@ I0926 14:05:56.173557   15312 clusterdeployer.go:124] Creating master  in namesp
 I0926 14:05:56.183134   15312 clusterclient.go:563] Waiting for Machine aws-controlplane-65vtk to become ready...
 ```
 
+#### Actuator logs
+After running clusterctl, a minikube instance is going to be running locally, which spins up an AWS cluster. The actuators run inside the minikube cluster and you can tail logs using minikube/kubectl. The actuators run inside the respective controllers, which are run in a single pod with prefix `clusterapi-controllers-....`.
+
+> Make sure minikube context is properly set `minikube update-context`
+
+- To tail cluster actuator logs, `kubectl logs -f clusterapi-controllers-<...> aws-cluster-controller`
```

Comment:
I use this command that does not require the user to know the actual pod name:

```
# for machine controller
 kubectl get po -o name | grep clusterapi-controllers | xargs kubectl logs -c aws-machine-controller -f
# for cluster controller
 kubectl get po -o name | grep clusterapi-controllers | xargs kubectl logs -c aws-cluster-controller -f
```

v1.12 kubectl  has some nicer features in it that we could use as well, but i'm not sure if those are client side or server side. It'd be a non-starter if it's server side since we're bound to 1.9.4 for now.